src/main/java/ifsul/agileproject/rachadinha/controller/RachaController.java

```diff
@@ -18,16 +24,25 @@
 @AllArgsConstructor
 public class RachaController {
 
-    private final RachaServiceImpl rachaService;
+	private final RachaServiceImpl rachaService;
+
+	//Criar racha
+	@PostMapping("/criar")
+	public ResponseEntity<RachaResponseDTO> createRacha(@RequestBody RachaRegisterDTO rachaDTO) {
+		Racha racha = rachaService.saveRacha(rachaDTO);
+
+		RachaResponseDTO rachaResponseDTO = RachaResponseDTO.transformarEmDto(racha);
 
-    //Criar racha
-    @PostMapping("/criar")
-    public ResponseEntity<RachaResponseDTO> createRacha(@RequestBody RachaRegisterDTO rachaDTO) {
-        Racha racha = rachaService.saveRacha(rachaDTO);
+		return new ResponseEntity<>(rachaResponseDTO, HttpStatus.CREATED);
+	}
 
-        RachaResponseDTO rachaResponseDTO = RachaResponseDTO.transformarEmDto(racha);
+	@GetMapping("/findByOwner/{id}")
+	public ResponseEntity<List<Racha>> findByOwner(@PathVariable Long id) {
```

Comment:
Maybe is better return a List of RachaRespondeDTO

Reply:
Return type changed to RachaResponseDTO